src/CodeGen_X86.cpp

```diff
@@ -600,6 +619,22 @@ void CodeGen_X86::visit(const Call *op) {
         }
     }
 
+    // Special case of saturating_pmulhrs.
```

Comment:
Maybe add a sentence about why it is a special case, as it's not immediately obvious to me

Reply:
Bleh, this was a bad comment. Basically we just want to generate `pmulhrs` when we see the saturating variant, plus a check for the case where saturation would trigger. I will update the comment.

Reply:
@steven-johnson is the comment added in [060ec46](https://github.com/halide/Halide/pull/7129/commits/060ec46c7251364e282758a7db8a4d164b17e7e7) sufficient?

Reply:
LGTM